shapedetector.py

```diff
@@ -1,5 +1,6 @@
-import cv2, imutils
-
+import cv2
+import imutils
+import numpy as np
 
 class ShapeDetector:
     def __init__(self):
```

Comment:
if youre just passing, you dont need this function, it will automatically be there.  you should only defined __init__ if you need to change how the constructor for the object works